### ISSUE TITLE
Renaming file name (F2) causing unexpected end.

### ISSUE THREAD

**Issue by marensas** [microsoft/vscode#113491]
Issue Type: <b>Bug</b>

1. Select file
2. Press F2 ( rename file )
3. Start renaming.
4. Few file letters filled up, renaming saved ???
5. Other file letters went to the file search.

Bug: it's seems like in step 4 somehow ENTER was pressed and filling was stopped. Clicking on file again and renaming does not cause this error.

VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:34:46.910Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.81GB (2.88GB free)|
|Process Argv|--crash-reporter-id f6e971cf-fe58-4a44-86d7-00fad1d676ac|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|12.0.1
jest-snippets|and|1.8.0
vscode-intelephense-client|bme|1.5.4
gitlens|eam|11.1.0
prettier-vscode|esb|5.8.0
php-debug|fel|1.14.5
php-namespace-resolver|Meh|1.1.8
vscode-docker|ms-|1.9.0
php-docblocker|nei|2.1.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221877
python763:30178808
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
python504:30227505
vswsl492:30211401
wsl2promptcf:30224613
pythonvsdeb440cf:30237828
unusedprompt:30224610
folderexplorer:30224614
openfilemenucf:30224648
pythonvsded773cf:30236630

```

</details>

**Comment by stnmtz** [comment]
You were faster - this one is making me crazy. Also happens on a virgin vscode in an empty folder without any version management.

Waiting for about 2 seconds after copying and pasting a file and before pressing F2 to rename the new file helps also. Anyhow, I keep stepping into this issue multiple times a day. 

I noticed that the explorer UI seems to make a kind of refresh 1-2 seconds after copying the file (the tiny vertical "folder" line to the left flashes). After the flash pressing F2 is ok without losing the focus. Renaming the new file by mouse (right-click, rename) works - but I guess because navigating via the context menu is just too slow (I see the flash while the context menu is open).

The behavior is a little bit better with `"workbench.list.automaticKeyboardNavigation": false` - as the keyboard filtering doesn't automatically fetch half of the typed-in characters after the rename input loses focus. 

Bug was introduced with the latest update as far as I can tell.

**Comment by ArturoDent** [comment]
Can you check if this is a duplicate of https://github.com/microsoft/vscode/issues/111652 and fixed in the Insiders' Build?

**Comment by stnmtz** [comment]
Yes, the problem is fixed in insiders build (1.53.0-insider) and it seems to be a duplicate of #111652.

**Comment by isidorn** [comment]
Thanks for checking. And sorry about the bug.
